config/kloubak2-subt-estop-lora-jetson.json

```diff
@@ -225,7 +231,9 @@
               ["gas_serial.raw", "gas_detector.raw"],
               ["gas_detector.raw", "gas_serial.raw"],
               ["kloubak.angle", "to_jetson_rear.angle"],
-              ["localization.pose3d", "to_jetson_rear.pose3d"]
+              ["localization.pose3d", "to_jetson_rear.pose3d"],
+              ["kloubak.angle", "to_jetson_front.angle"],
```

Comment:
As far as I can tell, the `kloubak` driver currently does not produce an `angle` message. You need to create it and it needs to be in a form of a pose, i.e. `[pose, quaternion]`. In this case, pose should always be `[0, 0, 0]` and when the robot is straight, the quaternion should represent a 180deg yaw. Roll and pitch should be, unless the joint is more flexible than I remember, also always zero.